src/Core/Form/IdentifiableObject/CommandBuilder/Product/OptionsCommandBuilder.php

```diff
@@ -38,11 +38,12 @@ final class OptionsCommandBuilder implements ProductCommandBuilderInterface
      */
     public function buildCommand(ProductId $productId, array $formData): array
     {
-        if (!isset($formData['options'])) {
+        if (!isset($formData['options']) && !isset($formData['manufacturer']['manufacturer_id'])) {
```

Comment:
```suggestion
        if (!isset($formData['options'], $formData['manufacturer']['manufacturer_id'])) {
```
If it's an `and` you can do it :sweat_smile: